dictionary.py

```diff
@@ -1,12 +1,14 @@
-#variable for the empty dictionary
-sq_number = dict()
-
 #function to generate the dictionary keys and values
-def dictinary():
-    #create dictionary keys as numbers with values as their squares
-    for n in range(1,16):
-        sq_number[n] = n**2
-    return sq_number
+def dictinary(a,b):
+    if type(a) == int and type(b) == int:
+        #variable for the empty dictionary
+        sq_number = dict()
```

Comment:
Would appreciate if you can reduce lines 5-9 into comprehension. You can read about comprehension [here](https://medium.com/the-andela-way/a-cheap-introduction-to-comprehension-in-python-2269895f996f)

Reply:
@dennisja , When I read through the concept and it's really good, I have manged to use it in the recent changes.